[0m11:48:29.382402 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024ADEAB2360>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024ADF23FC50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024ADF23CFE0>]}


============================== 11:48:29.398773 | a55f6622-2f95-4602-8718-cf9435b19309 ==============================
[0m11:48:29.398773 [info ] [MainThread]: Running with dbt=1.9.0
[0m11:48:29.401779 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\luizf\\.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init vendas_dbt', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m11:48:29.558388 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'WindowsPath'.. Dictionary: {'dir': WindowsPath('C:/Users/luizf/.dbt')}
[0m11:48:29.562442 [info ] [MainThread]: Creating dbt configuration folder at 
[0m11:48:29.568981 [debug] [MainThread]: Starter project path: D:\jornada_de_dados\datawarehousezapflow\.venv\Lib\site-packages\dbt\include\starter_project
[0m11:48:29.603136 [info ] [MainThread]: 
Your new dbt project "vendas_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:48:29.604118 [info ] [MainThread]: Setting up your profile.
[0m11:52:53.121081 [info ] [MainThread]: Profile vendas_dbt written to C:\Users\luizf\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m11:52:53.124173 [debug] [MainThread]: Command `dbt init` succeeded at 11:52:53.123161 after 264.02 seconds
[0m11:52:53.124173 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024ADC0AA0C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024ADEFE0B60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000024ADF913D40>]}
[0m11:52:53.125163 [debug] [MainThread]: Flushing usage events
[0m11:52:53.751802 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m11:14:25.705719 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002435AD2D730>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002435B6F48C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002435B9CF920>]}


============================== 11:14:25.770545 | ef32c645-d596-4423-b128-be6a5d3ae83b ==============================
[0m11:14:25.770545 [info ] [MainThread]: Running with dbt=1.9.0
[0m11:14:25.772501 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\luizf\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m11:14:25.773548 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\jornada_de_dados\datawarehousezapflow\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m11:14:25.779050 [debug] [MainThread]: Command `dbt run` failed at 11:14:25.778028 after 0.33 seconds
[0m11:14:25.780036 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002435AC0D970>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002435DFD83E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002435AD2D730>]}
[0m11:14:25.781039 [debug] [MainThread]: Flushing usage events
[0m11:14:27.152468 [debug] [MainThread]: An error was encountered while trying to flush usage events
